src/main/java/com/denys/toukdemo/helper/RoomHelper.java

```diff
@@ -0,0 +1,53 @@
+package com.denys.toukdemo.helper;
+
+import com.denys.toukdemo.dto.ReservationUnit;
+import com.denys.toukdemo.dto.Room;
+
+import java.util.*;
+
+public class RoomHelper {
+    private Room room;
+
+    public RoomHelper(Room room) {
+        this.room = room;
+    }
+
+    public boolean hasSinglePlaceBetweenTwoAlreadyReserved(List<ReservationUnit> units){
+        Map<Integer, List<Integer>> uniqueRows = createMapUniqueRows(units);
+        for(Map.Entry<Integer, List<Integer>> pair : uniqueRows.entrySet()){
```

Comment:
czy tych forów nie da się napisać lepiej w nowych javach?

Reply:
> czy tych forów nie da się napisać lepiej w nowych javach?

Jak już napisałem wczesznej żle by to wygłądało na razie. Kiedyś dam radę fajnie używać stream api